Automattic/woocommerce-payments
# add position absolute to popup

Fixes #2698



#### Changes proposed in this Pull Request

This PR adds position absolute to popup on woocommerce payments page in order to fix the misaligned position on Safari .

Popup before changes: 


<img width="739" alt="Screenshot 2021-10-04 at 13 05 36" src="https://user-images.githubusercontent.com/82826872/135832943-624fea85-5dbe-48fd-b320-89310dc37903.png">
Popup after changes:


<img width="1418" alt="Screenshot 2021-10-04 at 13 05 54" src="https://user-images.githubusercontent.com/82826872/135832915-f69c2760-d5fb-46d1-8c60-a73880c49f9d.png">






#### Testing instructions





* Open Wp admin in Safari ( v14.1.2 )
* Go to Payments-> Overview
* Click on Dismiss button from "Sell worldwide in multiple currencies" section
* Check if pop-up is properly aligned

-------------------


**Post merge**
